### ISSUE TITLE
luci-app-wifischedule: Runtime error / unable to access luci after updating packages using Attended Sysupgrade.

### ISSUE THREAD

**Issue by ghost** [openwrt/luci#7390]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### screenshots or captures

![luci-app-wifischedule](https://github.com/user-attachments/assets/c2eb4ce9-10d8-46f2-a984-d60bb756279e)

### Actual behaviour

As seen from the screenshot above.
I could not access luci from browser.
The error occurred after I updated many packages using Attended Sysupgrade.

### Expected behaviour

I should be able to access luci from browser.

Solution:
Issue resolved by removing **luci-app-wifischedule** and keeping dependencies (luci-compat).

### Steps to reproduce

1. Type router ip address
2. Enter

### Additional Information

```shell
NAME="OpenWrt"
VERSION="23.05.5"
ID="openwrt"
ID_LIKE="lede openwrt"
PRETTY_NAME="OpenWrt 23.05.5"
VERSION_ID="23.05.5"
HOME_URL="https://openwrt.org/"
BUG_URL="https://bugs.openwrt.org/"
SUPPORT_URL="https://forum.openwrt.org/"
BUILD_ID="r24106-10cc5fcd00"
OPENWRT_BOARD="ipq40xx/generic"
OPENWRT_ARCH="arm_cortex-a7_neon-vfpv4"
OPENWRT_TAINTS=""
OPENWRT_DEVICE_MANUFACTURER="OpenWrt"
OPENWRT_DEVICE_MANUFACTURER_URL="https://openwrt.org/"
OPENWRT_DEVICE_PRODUCT="Generic"
OPENWRT_DEVICE_REVISION="v0"
OPENWRT_RELEASE="OpenWrt 23.05.5 r24106-10cc5fcd00"

luci openwrt-23.05 branch (git-24.313.38121-76e4eca)

luci-app-wifischedule git-24.313.38121-76e4eca
```

**Comment by Ramon00** [comment]
I think the "wifischedule" on 23.05 is incompatible with "wifischedule" on master. So the luci-app-wifischedule should not have been backported to 23.05 @systemcrash can you revert?

**Comment by Ramon00** [comment]
Also it seems on 23.05 there is now both js and lua code for the app. For sure that will not work.

**Comment by systemcrash** [comment]
Bizzare.

**Comment by systemcrash** [comment]
Closed by cd1e372bb036286e3d9519805a79af37bed3c77a
